src/wyzeapy/wyze_auth_lib.py

```diff
@@ -18,32 +19,46 @@
 
 
 class Token:
-    def __init__(self, access_token, refresh_token, last_login_time):
+    # Token is good for 216,000 seconds (60 hours) but 48 hours seems like a reasonable refresh interval
+    REFRESH_INTERVAL = 172800
+
+    def __init__(self, access_token, refresh_token):
         self.access_token: str = access_token
         self.refresh_token: str = refresh_token
-        self.last_login_time: float = last_login_time
+        self.refresh_time: float = time.time() + Token.REFRESH_INTERVAL
+
+    @property
+    def access_token(self):
+        return self._access_token
+
+    @access_token.setter
+    def access_token(self, access_token):
+        self._access_token = access_token
+        self._refresh_time = time.time() + Token.REFRESH_INTERVAL
 
 
 class WyzeAuthLib:
     token: Optional[Token] = None
     _conn: TCPConnector
     _session: ClientSession
 
-    def __init__(self, username=None, password=None, token: Token = None):
+    def __init__(self, username=None, password=None, token: Token = None, token_callback=None):
```

Comment:
I thought this was just one callback that is stored in the client? Shouldn't it be called "token_callback"

Reply:
you're right. I actually had another commit below that fixes this.